src/account/operations/syncing/addresses.rs

```diff
@@ -208,65 +208,138 @@ impl AccountHandle {
         address: AddressWithUnspentOutputs,
         sync_options: &SyncOptions,
     ) -> crate::Result<(AddressWithUnspentOutputs, Vec<OutputId>)> {
-        // Get basic outputs
-        let mut output_ids = client
-            .basic_output_ids(vec![QueryParameter::Address(address.address.to_bech32())])
-            .await?;
-        // TODO: update when https://github.com/iotaledger/inx-indexer/issues/33 gets fixed or if that's not happening, send requests in parallel
-        // Get outputs where the address is in the storage deposit return unlock condition
-        let sdr_output_ids = client
-            .basic_output_ids(vec![QueryParameter::StorageReturnAddress(address.address.to_bech32())])
-            .await?;
-        output_ids.extend(sdr_output_ids.into_iter());
-        // Get outputs where the address is in the expiration unlock condition
-        let expiration_output_ids = client
-            .basic_output_ids(vec![QueryParameter::ExpirationReturnAddress(
-                address.address.to_bech32(),
-            )])
-            .await?;
-        output_ids.extend(expiration_output_ids.into_iter());
+        // let mut tasks = Vec::new();
+        let client_1 = client.clone();
```

Comment:
Can you move these to the actual blocks where they are needed please ? You can then drop the suffixes

Reply:
How? Because of async move { Client is moved if we don't do it before

Reply:
```rs
            // Get basic outputs
            async move {
                let client = client.clone();
                tokio::spawn(async move {
```
This should work